files/zh-cn/web/javascript/reference/global_objects/string/anchor/index.md

```diff
@@ -3,51 +3,43 @@ title: String.prototype.anchor()
 slug: Web/JavaScript/Reference/Global_Objects/String/anchor
 ---
 
-{{JSRef("Global_Objects", "String")}}
+{{JSRef}} {{deprecated_header}}
 
-## 概述
+{{jsxref("String")}} 的 **`anchor()`** 方法创建一个字符串，将该字符串嵌入到一个带有名称的 {{HTMLElement("a")}} 元素中（`<a name="...">str</a>`）。
```

Comment:
The anchor() method of [String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String) values creates a string that embeds this string in an [<a>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) element with a name (<a name="...">str</a>).
```suggestion
{{jsxref("String")}} 的 **`anchor()`** 方法创建一个带有名称的 {{HTMLElement("a")}} 元素字符串，其中嵌入了调用的字符串（`<a name="...">str</a>`）。
```